### DISCUSSION TITLE
Seit iOS 16.0 klingeln die HomePods nicht mehr

### DISCUSSION THREAD

**Discussion by gerd2005** [thkl/hap-homematic#581] [General]
Hallo,

ich habe einen HMIP-DSD-PCB, der bislang dafür gesorgt hat, dass beim betätigen der Klingel eine Nachricht auf die iPhones kommt und meine HomePods mini bimmeln. Letzteres funktioniert (gefühlt) seit dem iOS Update auf 16.0 nicht mehr. Hat noch jemand ein ähnliches Problem?

**Comment by blackbasket** [comment]
Hey Gerd,

also ich nutze die "normale" HmIP-Klingel. Da kann ich das Verhalten nicht bestätigen. Allerdings hatte ich das in den letzten Jahren immer mal wieder nach einem OS-Update. Mit einem reboot der RM + Apple-Komponenten hat es sich manchmal lösen lassen. Sonst hat nur warten auf das nächste OS-Update geholfen.

Lg,
Marcel

**Reply by gerd2005** [reply]
Hallo, was meinst du mit „normale“ HMIP-Klingel?

**Comment by blackbasket** [comment]
Siehe #617

**Comment by ZenitES** [comment]
Hallo @gerd2005,

hast du zufällig eine Lösung gefunden? Habe auch HMIP-DSD-PCB. Habe auch schon alles neu eingerichtet und wir haben bereits schon iOS 16.4 aber meine Minis klingeln immer noch nicht.

**Reply by gerd2005** [reply]
Hi,

Nein die klingeln immer noch nicht. Ich glaube, dass auch keine Push-Nachrichten mehr kommen.
